services/githubService.js

```diff
@@ -106,6 +106,25 @@ function getFetch(url) {
 function getFetchWithAuthToken(url, params = null, headers = null) {
   return utils.fetch(url, "get", params, null, headers);
 }
+
+/**
+ * Fetches the user github informations by userId using GitHub API
+ * @param githubUsername {string} - githubUsername String
+ */
+
+const fetchUserGithubInfo = async (githubUsername) => {
+  try {
+    if (!githubUsername) {
+      throw new Error("github username missing");
+    }
```

Comment:
if block will throw one error that will be caught in catch block and catch block will send another message for this condition logger wont be abe to log error thus we will get errors. i.e. here the error.response.data.message will be undefined.

use separate try catch

Reply:
Ok next time I will try to  follow these steps